tests/test_simple_harvest.py

```diff
@@ -95,8 +105,8 @@ def test_simple_harvest(
 
     new_assets = vault.totalAssets()
 
-    # confirm we made money
-    assert new_assets > old_assets
+    # confirm we made money, or at least that we have about the same
+    assert new_assets >= old_assets
```

Comment:
changed to '>=' to include the scenario in which the gauge has no CRV to distribute